mono/metadata/threads.c

```diff
@@ -2553,6 +2553,9 @@ mono_thread_internal_abort (MonoInternalThread *thread, gboolean appdomain_unloa
 void
 ves_icall_System_Threading_Thread_ResetAbort (MonoThreadObjectHandle this_obj, MonoError *error)
 {
+	// FIXME Why are ves_icall_System_Threading_Thread_ResetAbort and mono_thread_internal_reset_abort so different?
+	// FIXME? this_obj is unused but passed by reference source.
```

Comment:
`System.Threading.Thread.ResetAbortNative` is private so we could make it static, I suppose...